dist/img/speech-bubble-line.svg

```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" width="14.903" height="14.427" viewBox="0 0 14.903 14.427"><defs><style>.a{fill:#a3a3a3;stroke:#a3a3a3;stroke-width:0.5px;}</style></defs><path class="a" d="M39.971.574H32.735a2.774,2.774,0,0,0-2.929,2.567V7.672a2.775,2.775,0,0,0,2.929,2.567H35.01a5.325,5.325,0,0,1-1.924,3.079,9.627,9.627,0,0,0,5.386-3.079h.393c1.611,0,4.035-1.155,4.035-2.567V3.141A2.774,2.774,0,0,0,39.971.574Zm-.357,4.068a1,1,0,0,0-1.063.932,1.072,1.072,0,0,0,2.126,0A1,1,0,0,0,39.614,4.642Zm-6.522,0a1,1,0,0,0-1.063.932,1.072,1.072,0,0,0,2.126,0A1,1,0,0,0,33.091,4.642Zm3.261,0a1,1,0,0,0-1.063.932,1.072,1.072,0,0,0,2.126,0A1,1,0,0,0,36.353,4.642ZM32.735,0h7.237V0a3.942,3.942,0,0,1,1.937.5,3.539,3.539,0,0,1,.593.42h0a2.953,2.953,0,0,1,1.051,2.218h0V7.672h0a2.9,2.9,0,0,1-1.708,2.29,6.39,6.39,0,0,1-2.979.848v0h0v0H38.79a9.484,9.484,0,0,1-2.279,1.822,11.569,11.569,0,0,1-3.264,1.242v0a.712.712,0,0,1-.65-.175.524.524,0,0,1,.056-.807A4.284,4.284,0,0,0,33.791,11.6a5.207,5.207,0,0,0,.356-.784H32.734v0a3.841,3.841,0,0,1-2.531-.92,2.953,2.953,0,0,1-1.05-2.219h0V3.141h0A2.941,2.941,0,0,1,30.167.957L30.2.922A3.841,3.841,0,0,1,32.734,0V0h0Z" transform="translate(-28.901 0.25)"/></svg>
```

Comment:
if you don't use this file, just remove it?

Reply:
finished